src/test/java/net/crusadergames/bugwars/controller/GameMapControllerIT.java

```diff
@@ -0,0 +1,168 @@
+package net.crusadergames.bugwars.controller;
+
+import net.crusadergames.bugwars.dto.request.GameMapRequest;
```

Comment:
Enhancement: Are there tests checking endpoints for this controller? These appear to be JUnit tests only.